memcache_proxy/dtog.go

```diff
@@ -33,6 +33,12 @@ import (
 	pb "google.golang.org/appengine/notreallyinternal/memcache"
 )
 
+const (
+	// Max App Engine Memcache value size (10^6 bytes).
+	// https://cloud.google.com/appengine/docs/python/memcache/
+	maxMemcacheValueSize = 1000000
```

Comment:
Java has the same limits as Go, so I think it's better to do this properly in a separate CL.